src/eei.cpp

```diff
@@ -585,7 +643,12 @@ namespace HeraVM {
       create_message.flags = 0;
 
       evm_result create_result;
+     
+      takeGas(create_message.gas);
+      takeGas(GasSchedule::create);
+      takeGas(GasSchedule::callnewacct);
```

Comment:
This is not needed here. However, changing GasSchedule::create to be equal GasSchedule::call + GasSchedule::callnewacct would make sense, @axic ?